base/entries.go

```diff
@@ -0,0 +1,63 @@
+package base
+
+import (
+	"fmt"
+
+	"github.com/qri-io/dataset/dsio"
+)
+
+// ReadEntries reads entries and returns them as a native go array or map
```

Comment:
Huge fan of moving this into one spot. Lovely de-duplication. In the long run we may want to put this directly in the dsio package. This is a great start.